pkg/scheduler/backend/queue/scheduling_queue.go

```diff
@@ -718,7 +718,7 @@ func (p *PriorityQueue) addUnschedulableWithoutQueueingHint(logger klog.Logger,
 		metrics.UnschedulableReason(plugin, pInfo.Pod.Spec.SchedulerName).Inc()
 	}
 	if p.moveRequestCycle >= podSchedulingCycle || len(rejectorPlugins) == 0 {
-		// Two cases to move a Pod to the active/backoff queue:
+		// Two cases to move a Pod to ctive/backoff queue:
```

Comment:
Revert the change here:
```suggestion
		// Two cases to move a Pod to the active/backoff queue:
```